_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -90,22 +90,24 @@ release candidates.*
   to the same address when only a subset of those inputs might be needed.
   For this reason, `avoidpartialspends` is disabled by default unless
   the `avoid_reuse` flag is enabled for the wallet (see [Newsletter
-  #52][news52 avoid_reuse]).  The new `maxapsfee` feature now gives
-  users a choice between three configurations:
+  #52][news52 avoid_reuse]).  It is for this default case that maxapsfee
```

Comment:
unsure if "maxapsfee" was intentionally left without code markdown in this line, but it's probably  fine like this as well